pyomo/repn/plugins/ampl/ampl_.py

```diff
@@ -87,12 +87,14 @@ def _build_op_template():
     div_comment = "\t#/"
     _op_template[EXPR.ProductExpression] = prod_template
     _op_comment[EXPR.ProductExpression] = prod_comment
+    _op_template[EXPR.DivisionExpression] = div_template
+    _op_comment[EXPR.DivisionExpression] = div_comment
     _op_template[EXPR.ReciprocalExpression] = div_template
     _op_comment[EXPR.ReciprocalExpression] = div_comment
-    del prod_template
-    del prod_comment
-    del div_template
-    del div_comment
+    #del prod_template
```

Comment:
Why were these commented out?

Reply:
I commented them out because I could not figure out why we were going to the effort of deleting them: I can't see how they would lead to memory leaks, nor why keeping a string template hanging around would impact how our code ran.